components/app/myrw/widgets/pages/new.js

```diff
@@ -184,29 +184,29 @@ class WidgetsNew extends React.Component {
 
 
   loadDatasets() {
-    this.datasetsService.fetchAllData({ filters: { published: true } }).then((response) => {
+    this.datasetsService.fetchAllData({ filters: { published: true }, includes: "metadata" }).then((response) => {
       this.setState({
         datasets: [...this.state.datasets, ...response.map(dataset => ({
           id: dataset.id,
           type: dataset.type,
           provider: dataset.provider,
           tableName: dataset.tableName,
-          label: dataset.name,
+          label: dataset.metadata[0] && dataset.metadata[0].attributes.info ? dataset.metadata[0].attributes.info.name : dataset.name,
           value: dataset.id
         }))],
         loadingPublishedDatasets: false
       });
     });
 
     this.datasetsService.fetchAllData(
-      { filters: { userId: this.props.user.id } }).then((response) => {
+      { filters: { userId: this.props.user.id }, includes: "metadata" }).then((response) => {
       this.setState({
         datasets: [...this.state.datasets, ...response.map(dataset => ({
           id: dataset.id,
           type: dataset.type,
           provider: dataset.provider,
           tableName: dataset.tableName,
-          label: dataset.name,
+          label: dataset.metadata[0] && dataset.metadata[0].attributes.info ? dataset.metadata[0].attributes.info.name : dataset.name,
           value: dataset.id
```

Comment:
Please fix Eslint issues such as use of double quote or lines that are too long. 
Also please use in this kind of cases something like:
```
const metadata = dataset.metadata[0];
label: metadata && metadata.attributes.info ? ....
```
so that you don't end up having huge lines